neuro_san/service/http/handlers/base_request_handler.py

```diff
@@ -27,9 +27,9 @@
 import tornado
 from tornado.web import RequestHandler
 
-from neuro_san.http_sidecar.logging.http_logger import HttpLogger
-from neuro_san.http_sidecar.interfaces.agent_authorizer import AgentAuthorizer
-from neuro_san.http_sidecar.interfaces.agents_updater import AgentsUpdater
+from neuro_san.service.http.interfaces.agent_authorizer import AgentAuthorizer
+from neuro_san.service.http.interfaces.agents_updater import AgentsUpdater
+from neuro_san.service.http.logging.http_logger import HttpLogger
```

Comment:
More imports.
@andreidenissov-cog : Below here is a dictionary of grpc status codes to http status codes.
It would be good to investigate if we really need that and any other reference to grpc in this http area any more.

Reply:
That's a good catch. We probably don't need it anymore. Not here at least.